bokeh-demo/examples/landing_page/main.py

```diff
@@ -0,0 +1,29 @@
+import pickle
+
+from bokeh.io import curdoc
+from bokeh.models.callbacks import CustomJS
+from bokeh.models.widgets import Button, TextInput
+
+from bokeh_demo.settings import settings
+
+text = TextInput(title="Text input", value="input your text here")
+
+
+def save_text(attr, old, new):
+    with open(settings.selected_pids_path, "wb") as f:
+        pickle.dump(text.value, f)
```

Comment:
Hate to be that guy, but

Are we sure we want to use pickle for this? It is generally considered an unsafe module, see https://docs.python.org/3.10/library/pickle.html.
I'm not sure exactly how, but one could conceive that someone would be able to hack the query to make us here pickle something unsafe. Maybe using `json` is better (as suggested in the above linked docs).